docs/source/testing/trip_test.rst

```diff
@@ -0,0 +1,99 @@
+    2. If your modification contains a bug fix, we want to know which test cases are affected. We also want to know which P/R resulted in the bug fix.
+    3. You may think your modifications won't have changed Isca's results, but we want to make sure that's the case. The trip tests are very good at highlighting unexpected consequences of your changes, and help us keep to a high-standard of reproducibility.
+
+
+How do I run the trip tests?
+---------------------------------
+
+To run the trip tests, navigate to ``${GFDL_BASE}/exp/test_cases/trip_test/``. From this directory the trip tests are run using the command-line interface. The simplest example call is
+::
+    ./trip_test_command_line 155661f ec29bf3
+This would run all of the test cases, comparing the results using commit ID ``155661f`` with commit ID ``ec29bf3``.
+
+The optional arguments are
+::
+    -e 'all' - Runs all test experiments
+    -n 4     - Uses 4 cores to run Isca
+    -r 'git@github.com:execlim/Isca' - Uses the online Isca repo to checkout commits from
+
+To specify a subset of the test cases, use the ``-e`` option
+::
+    ./trip_test_command_line 155661f ec29bf3 -e 'axisymmetric' 'bucket_model' 'frierson'
+
+To change the number of cores Isca is run on, use the ``-n`` option
+::
+    ./trip_test_command_line 155661f ec29bf3 -e 'axisymmetric' 'bucket_model' 'frierson' -n 4
+
+To specify the github repo used for the tests (e.g. your fork rather than ``Execlim/Isca``), use the ``-r`` option
+::
+    ./trip_test_command_line 155661f ec29bf3 -e 'axisymmetric' 'bucket_model' 'frierson' -r git@github.com:sit23/Isca
+
+**Please note that by default the trip tests will try to connect to the ExeClim/Isca repository using SSH**. If you use HTTPs please use the option ``-r https://github.com/ExeClim/Isca.git``.
+
+Example output
+--------------
+
+Running the command
+::
+    ./trip_test_command_line 155661f8c7945049cbac0dcf2019bb17fe7a6a8d ec29bf389cf5ac53b50b23c363040479a6392e52
+
+Produces the following summary output
+::
+    Results for all of the test cases ran comparing 155661f and ec29bf3 are as follows...
+    variable_co2_grey : pass
+    realistic_continents_fixed_sst : pass
+    realistic_continents_variable_qflux : pass
+    held_suarez : pass
+    variable_co2_rrtm : pass
+    bucket_model : fail
+    top_down_test : pass
+    frierson : pass
+    axisymmetric : pass
+    MiMA : pass
+    giant_planet : pass
+    Nightmare, some tests have failed
+
+The bucket test is the only one that fails in this instance. This is because the bucket model formulation was changed between these two commits, and so it is expected that the results with the two commits will differ. However, we note that the other tests have their results unchanged, meaning this is a safe modification to the code. Any unexpected failures should be investigated before submitting a pull-request.
```

Comment:
When you open your PR please copy and paste this summary output into the PR ticket.

Reply:
I've added a new section on specifically what to do for a pr. See what you think.